Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -279,18 +278,17 @@ public float AnimationLength
             set { animationLength = value; }
         }
 
-        [Tooltip("Number of columns or rows in respect to ViewableArea and ScrollDirection.")]
+        [Tooltip("Number of items in a line on up-down scroll direction or number of items in a column on left-right scroll direction.")]
         [SerializeField]
-        [Range(1, 500)]
-        private int tiers = 2;
+        private int itemsPerTier = 1;
```

Comment:
I think this needs a FormerlySerializedAs attribute to that previous versions will work (i.e. across upgrades)

Reply:
Actually since this is an experimental area I suppose it's not necessary, but it's fairly light weight and would probably help a consumer :)